src/components/dls/Skeleton/Skeleton.module.scss

```diff
@@ -23,18 +14,16 @@ $animationDuration: 8s;
 
 .active::before {
   border-radius: var(--border-radius-default);
-  background-image: linear-gradient(
-    270deg,
-    var(--shade-0),
-    var(--shade-1),
-    var(--shade-2),
-    var(--shade-1),
-    var(--shade-0)
-  );
-  background-size: 400% 100%; // increase background size's width to 4x to make background animate-able
-  animation: loading $animationDuration ease-in-out infinite;
-
-  content: "";
+
+  [data-theme="dark"] & {
+    background-color: var(--color-background-alternative-faint);
+  }
+  [data-theme="light"] & {
+    background-color: var(--color-secondary-faded);
+  }
+  animation: 1.5s ease-in-out 0.5s infinite normal none running loading;
```

Comment:
The speed value should be moved to constants e.g. `$transition-fast: 1.5s`. Also you might want to try using `var(--transition-fast)` which is `0.6s` instead of 0.5s.